docs/samples/custom/torchserve/logger/logger_demo.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>should the dumper deployed in the same namespace as inference service  in kfserving-test namespace?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/kfserving/pull/1228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>